qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/ComponentsAltActivity.kt

```diff
@@ -84,8 +87,18 @@ class ComponentsAltActivity : AppCompatActivity() {
             routeArrowInstallation = routeArrow(binding.mapView) {
                 options = customRouteArrowOptions()
             }
+            recenterButton(binding.mapView, binding.recenterButton) {
+                cameraOptions = CameraOptions.Builder()
+                    .zoom(10.0)
+                    .bearing(0.0)
+                    .build()
+                animationOptions = MapAnimationOptions.Builder()
+                    .duration(1000L)
+                    .build()
+            }
 
             // custom components
+            component(MapButtonsVisibility(binding.mapButtons))
```

Comment:
If my understanding https://github.com/mapbox/mapbox-navigation-android/pull/6158#discussion_r947357188 is correct, then this logic might not work. When we add `CameraModeButton` in the next PR, it will also hide/show based on `TripSessionState`, because camera mode button is a part of the same linear layout `binding.mapButtons`. However the camera mode button should always be visible no matter in whatever state you are in.

I think we should control the visibility of these buttons individually and not as a group because not all the buttons have the same requirement of when should they hide or show.

@tomaszrybakiewicz

Reply:
Resolving this based on https://github.com/mapbox/mapbox-navigation-android/pull/6158#issuecomment-1217357778